app/src/main/java/edu/ucandroid/weathernotice/ui/main/LocationLiveData.kt

```diff
@@ -0,0 +1,4 @@
+package edu.ucandroid.weathernotice.ui.main
+
```

Comment:
These three file changes were added in order to finish the implementation of Location services. Since they are considered dangerous permissions, the user should be prompted for permission to access. Obivously, this is just boilerplate and doesn't have any implementation quite yet, but can serve as a starting point for how you guys choose to implement it.

https://www.youtube.com/watch?v=VgZZemAwLTk&feature=youtu.be

Reply:
Right. we added the GPS before week 7 mod, and look at the Plant dairy example but at the decided to move forward with what we had  that the time. will revist.